portfolio/src/logo.svg

```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 841.9 595.3">
```

Comment:
@Ally4 This is applicable to all .svg files you have, put them into one folder, say assets/picture folder in order to have a clear structure.